jib-core/src/main/java/com/google/cloud/tools/jib/image/ImageReference.java

```diff
@@ -198,7 +198,7 @@ public static boolean isValidRepository(String repository) {
    * @return {@code true} if is a valid tag; {@code false} otherwise
    */
   public static boolean isValidTag(String tag) {
-    return tag.matches(TAG_REGEX) || tag.matches(DescriptorDigest.DIGEST_REGEX);
+    return tag != null && (tag.matches(TAG_REGEX) || tag.matches(DescriptorDigest.DIGEST_REGEX));
```

Comment:
`tag` isn't annotated with `@NotNull`, so checking null here doesn't seem pretty. Perhaps we should ensure `tag` is not null.

Reply:
Oops, `@Nullable` I mean.